.travis.yml

```diff
@@ -17,27 +17,29 @@ script:
   - |
     nix-env -i ShellCheck
     grep -RHL --exclude-dir node_modules node src/*/libexec | xargs shellcheck
-  - cachix push --watch-store dapp &
+  - [[ "$TRAVIS_REPO_SLUG" == "dapphub/dapptools" ]] && cachix push --watch-store dapp &
   - |
     PLATFORM=linux
 
-    if [[ "$TRAVIS_OS_NAME" == "osx" ]]; then PLATFORM=darwin; fi
+    [[ "$TRAVIS_OS_NAME" == "osx" ]] && PLATFORM=darwin
 
     # the additional call to `cachix push` is there to prevent a race condition
     # that happens when the above daemon is killed mid-push at the end of this
     # script.
     nix-build -j auto release.nix -A dapphub.$PLATFORM.stable | cachix push dapp
```

Comment:
Do we need to check the `REPO_SLUG` for this push as well?

Reply:
Yep.